### DISCUSSION TITLE
The consumption performance of local and remote rabbitmq IP addresses varies greatly

### DISCUSSION THREAD

**Discussion by Summer-Ameng** [rabbitmq/rabbitmq-server#8979] [Questions]
When I use localhost, ack can reach 10,000 or more, but when I use extranet IP, ack can only hover in the hundreds. Is there any configuration that needs to be adjusted?

**Comment by mkuratczyk** [comment]
What do you mean by "ack can reach 10,000"?

It's no surprise that a network introduces latency and impacts the throughput but whether what you see is normal or excessive difference is hard to tell without further details.

Ideally, please run https://perftest.rabbitmq.com/ and simulate your workload to see how much it differs.

**Reply by mkuratczyk** [reply]
perf-test is written in Java but that's not important - it's an AMQP client which can simulate all kinds of scenarios and provides metrics (to the console and/or Prometheus).

I still don't understand "ack can reach 10000". Please provide a screenshot to explain what metric you're talking about

**Reply by Summer-Ameng** [reply]
I used perf-test to test, it should be caused by network problems, I probably should not use the ack on the UI as an indicator of speed, thank you very much!

**Reply by michaelklishin** [reply]
Local connections enjoy the obvious data locality benefits, plus potential kernel-level optimizations compared to remote connection. It can have a very significant impact on throughput and latency.
